resources/bin/bal

```diff
@@ -30,11 +30,29 @@ elif test -d "$CURRENT_PATH/../dependencies/jdk8u202-b08-jre"; then
   JAVA_COMMAND="$CURRENT_PATH/../dependencies/jdk8u202-b08-jre/bin/java"
 fi
 
+if [ "$1" == "completion" ]
+then
+    if [ "$2" == "bash" ]
+    then
+      printf "#!/usr/bin/env bash\n\n"
+      cat $CURRENT_PATH/../scripts/bal_completion.bash
+    else
```

Comment:
@suganyasuven  shall we check zsh specifically? For others lets include invalid command